server/data/courseClient.test.ts

```diff
@@ -17,11 +17,15 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
   })
 
   const allCourses = courseFactory.buildList(3)
+  const course = courseFactory.build({
+    id: '28e47d30-30bf-4dab-a8eb-9fda3f6400e8',
+  })
```

Comment:
Do we need to make the ID in the course factory `params.id || ` for this to work (we do that for some other fields)?

Reply:
It appears not, and it looks like we can remove this from the `course` and `courseAudience` factories (I think we said we only need it when the `||` statement is in a `const`